pypesto/engine/multi_process.py

```diff
@@ -42,7 +43,7 @@ def __init__(self, n_procs: int = None):
                 f"appropriate on some systems.")
         self.n_procs: int = n_procs
 
-    def execute(self, tasks: List[Task]):
+    def execute(self, tasks: List[Task], progress_bar: bool = False):
```

Comment:
why is the default false here and in other places true?

Reply:
good catch